src/main/java/org/railway/HomePage.java

```diff
@@ -0,0 +1,15 @@
+package org.railway;
+
+import org.openqa.selenium.By;
+import org.railway.utils.Action;
+
+public class HomePage extends BasePage{
+    private By welcomeMessage = By.xpath("//strong[normalize-space()='%s']");
+
+    public boolean checkWelcomeMessage(String expectedMessage) {
```

Comment:
you should review all the methods with the prefix `check`, they do not check anything.